### PR TITLE
Rebrand

### PR DESCRIPTION
**Pull request by JaimeRZP**

### CODE REVIEW

**Comment by JaimeRZP** [comment]
@damonge @carlosggarcia any idea why installing the python deps might be failing?

**Comment by carlosggarcia** [comment]
yes. install python in the ci.yml

**Comment by JaimeRZP** [comment]
added it to the reqs which the CI installs

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Welcome to COSMOTECA's documentation!
 =====================================
 
-COSMOTECA is a public repository and pipeline that computes large scale
+COSMOTHECA is a public repository and pipeline that computes large scale
```
Comment (damonge) [inline docs/source/index.rst:4]: COSMOTHEKA

**Review by damonge — APPROVED** [review]
Just one typo. Fix and then feel free to merge when the tests pass.

File: docs/source/index.rst (hunk 1 above)
Comment (JaimeRZP) [reply docs/source/index.rst]: and above!

**Pull request merged**